lib/DidKeyDriver.js

```diff
@@ -3,29 +3,54 @@
  */
 import * as didIo from '@digitalbazaar/did-io';
 import {
-  getDid, getKey, getKeyAgreementKeyPair, setVerificationKeyPairId
+  getDid, getKey, getKeyAgreementKeyPair, getMultiKeyHeader,
+  setVerificationKeyPairId
 } from './helpers.js';
 import {
   Ed25519VerificationKey2020
 } from '@digitalbazaar/ed25519-verification-key-2020';
 
+const Ed25519Header = 'z6Mk';
+
 export class DidKeyDriver {
-  /**
-   * @param {object} options - Options hashmap.
-   * @param {object} [options.verificationSuite=Ed25519VerificationKey2020] -
-   *   Key suite for the signature verification key suite to use.
-   */
-  constructor({verificationSuite = Ed25519VerificationKey2020} = {}) {
+  constructor() {
     // used by did-io to register drivers
     this.method = 'key';
-    this.verificationSuite = verificationSuite;
+    this.allowedKeyTypeHandlers = new Map();
+    // Set default keyTypeHandler for Ed25519  multikey header type.
+    this.allowedKeyTypeHandlers.set(
+      Ed25519Header, Ed25519VerificationKey2020
+    );
```

Comment:
I don't think we want or need a default anymore. We should decouple entirely so `did-method-key` doesn't depend on any particular key library. The examples should show how to use various key types.